src/unix/udp.c

```diff
@@ -230,6 +229,15 @@ static int uv__udp_recvmmsg(uv_udp_t* handle, uv_buf_t* buf) {
                       msgs[k].msg_hdr.msg_name,
                       flags);
     }
+
+    if (handle->recv_cb != NULL) {
+      chunk_buf = uv_buf_init(iov[k].iov_base, iov[k].iov_len);
```

Comment:
Is the `k` as index here correct? Should it perhaps be `0`?

Reply:
indeed